### ISSUE TITLE
vanish

### ISSUE THREAD

**Issue by anhtlauncher** [0xVienna/Celery#9]
- when execute the script, it run normally but when i tab in and tab out of the game and try to click celery on the taskbar it just wont appear. Happened multiple times

too lazy to make a video

**Comment by 0xVienna** [comment]
Maybe one day we can decompile this and maybe we can modify. Credits to Ariez.

**Comment by 0xVienna** [comment]
WAIT REINJECT THINGY DOESNT NEED IT ALREADY HAS IT but it may just bug out.

**Comment by anhtlauncher** [comment]
> Maybe one day we can decompile this and maybe we can modify. Credits to Ariez.

🗿 lmfao

**Comment by 0xVienna** [comment]
Yep haha

**Comment by Tyler9862** [comment]
Ive already decompiled this my discord is fog#6399

**Comment by 0xVienna** [comment]
ACCEPT NOW

**Comment by anhtlauncher** [comment]
> Ive already decompiled this my discord is fog#6399

ACCEPT MINE IS anhtlauncher#5823

**Comment by 0xVienna** [comment]
It says that ur username not there

**Comment by HandSomeNissan** [comment]
this IS open source, go to .gg/celery (this thing is fake, he didnt actually make this)

**Comment by 0xVienna** [comment]
Yeah I just put credits, the devs of celery allow me if I put credits less go!,

**Comment by HandSomeNissan** [comment]
oh damn he actually finally did it
